elytron/src/main/java/org/wildfly/extension/elytron/ElytronExtension.java

```diff
@@ -75,8 +75,9 @@ public class ElytronExtension implements Extension {
     static final ModelVersion ELYTRON_16_0_0 = ModelVersion.create(16);
     static final ModelVersion ELYTRON_17_0_0 = ModelVersion.create(17);
     static final ModelVersion ELYTRON_18_0_0 = ModelVersion.create(18);
+    static final ModelVersion ELYTRON_19_0_0 = ModelVersion.create(19);
```

Comment:
OK - it looks like ModelVersion 18.0.0 was associated with WF 29-31, meaning that we need a new ModelVersion 19.0.0 to identify the model changes for WF32 and later, correct?
(I'm trying to make sure I understand the context of this change)

Reply:
@pferraro Yes that's correct. The context of this change is just to fix the fact that I did not do the subsystem model bump when I added a resource to the community stability level previously.